jerry-core/config.h

```diff
@@ -39,6 +39,7 @@
 # define CONFIG_DISABLE_ES2015_ARROW_FUNCTION
 # define CONFIG_DISABLE_ES2015_BUILTIN
 # define CONFIG_DISABLE_ES2015_CLASS
+# define CONFIG_DISABLE_ES2015_FUNCTION_PARAMETER_INITIALIZER
```

Comment:
This new configuration macro should be documented in jerry-core/profiles/README.md (around line 90).